frappe/public/js/frappe/form/layout.js

```diff
@@ -72,7 +72,7 @@ frappe.ui.form.Layout = class Layout {
 			reqd: 1,
 			hidden: 1,
 			label: __('Name'),
-			get_status: function(field) {
+			get_status: function (field) {
```

Comment:
Avoid formatting in unchanged lines or at least put it in a separate commit. 

It makes it difficult to review actual changes and refer to history in future.

Reply:
Can you undo these changes please?

Reply:
Yeah I will. Sorry for that. My editor must have formatted it automatically.